lookout/style/format/benchmarks/evaluate_smoke.py

```diff
@@ -328,14 +327,11 @@ def evaluate_smoke_entry(inputpath: str, reportdir: str, database: str, bblfsh:
                     repopath = inputpath / row["repo"]
                     config_json = {
                         SmokeEvalFormatAnalyzer.name:
-                            merge_dicts(analyze_config, {
+                            merge_dicts(config, {"common": {
```

Comment:
IDK the specifics here, but the things like repository name should be excluded from all the configuration. The only source of truth is what is embedded into the model file.

Reply:
Yes, it is posible, let's move it out of this PR https://github.com/src-d/style-analyzer/issues/651